package.json

```diff
@@ -53,20 +57,21 @@
     "babel-cli": "^6.18.0",
     "babel-core": "^6.21.0",
     "babel-eslint": "^7.1.1",
-    "babel-jest": "^18.0.0",
+    "babel-jest": "^20.0.3",
     "babel-plugin-syntax-object-rest-spread": "^6.13.0",
     "babel-plugin-transform-object-rest-spread": "^6.20.2",
     "babel-preset-es2015": "^6.18.0",
     "eslint": "^3.13.1",
     "eslint-config-airbnb": "^14.0.0",
     "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jest": "^20.0.3",
```

Comment:
I don't fully know if this is needed, as most of these are done during dev and caught up during PR reviews/submits:
![skjermbilde 2017-06-25 kl 19 52 01](https://user-images.githubusercontent.com/16735925/27518411-d75adb02-59df-11e7-8d6d-d99107395851.png)